tls/s2n_server_extensions.c

```diff
@@ -73,10 +72,15 @@ int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         total_size += s2n_kex_server_extension_size(conn->secure.cipher_suite->key_exchange_alg, conn);
     }
 
-    if (s2n_server_can_send_sct_list(conn)) {
+    if (s2n_server_can_send_ocsp(conn) && !is_tls13_conn) {
+        total_size += 4;
+    }
+
+    if (s2n_server_can_send_sct_list(conn) && !is_tls13_conn) {
```

Comment:
When moving the extensions to their own files, will the is_tls13_conn check remain here? Or will the extension be in control of which contexts it can appear in?

Reply:
is_tls13_conn will remain outside so the extensions can be shared across context